leftwm/src/bin/leftwm-worker.rs

```diff
@@ -4,8 +4,7 @@ use slog::{o, Drain};
 use std::panic;
 
 fn main() {
-    //let _log_guard = setup_logfile();
-    #[cfg(feature = "logging")]
+    // let _log_guard = setup_logfile();
```

Comment:
```suggestion
    #[cfg(feature = "log-to-file")]
    let _log_guard = setub_logfile();
```
and of course add this feature to `leftwm/Cargo.toml`